### ISSUE TITLE
Fix documentation relative to CSRF

### ISSUE THREAD

**Issue by charlax** [flask-admin/flask-admin#1136]
It looks like the documentation about CSRF is outdated. The doc says:

```python
from flask_admin.form import SecureForm
```

which raises an `ImportError` because SecureForm is not defined anywhere in flask-admin:

```
ImportError: cannot import name SecureForm
```

Checking the CSRF documentation, it looks like the way to do this has changed a little bit: https://wtforms.readthedocs.org/en/latest/csrf.html

I had a custom CSRF validator and generator, so I have a slightly more complicated solution, but wanted to post it here:

```python
from wtforms.csrf.core import CSRF
from flask.ext.admin import form

from my_package import csrf  # custom csrf implementation

# This is based on https://wtforms.readthedocs.org/en/latest/csrf.html
class MyCSRFAdapter(CSRF):
    def setup_form(self, form):
        self.csrf_context = form.meta.csrf_context
        return super(ClayCSRFAdapter, self).setup_form(form)

    def generate_csrf_token(self, csrf_token):
        return csrf.create_token()

    def validate_csrf_token(self, form, field):
        # Validation is done by before_request handler on the custom csrf implementation
        pass


class SecureForm(form.BaseForm):
    class Meta:
        csrf = True
        csrf_class = MyCSRFAdapter


# This is based on http://flask-admin.readthedocs.org/en/latest/advanced/#enabling-csrf-protection
class ModelView(BaseModelView):
    form_base_class = SecureForm


# Your code follows
class AdminView(ModelView):
    ...
```

I've verified that this works.

**Comment by pawl** [comment]
https://github.com/flask-admin/flask-admin/search?utf8=✓&q=SecureForm

SecureForm is in the master branch, but hasn't made it to any releases yet. We probably need to fix the docs so they show the most recent release instead of master.

**Comment by charlax** [comment]
Did you close this meaning this is fixed? I don't see a commit on master actually fixing the docs... I'm fine taking a stab at it if you want.

**Comment by pawl** [comment]
There should be a release soon to make it less of a problem.

I think @petrus-jvrensburg knows more about how changes to the documentation are deployed to http://flask-admin.readthedocs.org/ and he might be able to help fix this one.

**Comment by pawl** [comment]
@charlax This issue was fixed by the release of 1.4.0.

Mind if we close this one and open another issue that's specifically about locking the documentation on flask-admin.readthedocs.org to the latest release rather than what's on the master branch?

**Comment by charlax** [comment]
For sure!
